app/views/ProfileView/index.js

```diff
@@ -180,7 +185,26 @@ class ProfileView extends React.Component {
 
 		// Username
 		if (user.username !== username) {
-			params.username = username;
+			if (!username.match(/[$&+,:;=?@#|'.<>^* ()%!]/g)) {
+				params.username = username;
```

Comment:
You can't use this regex here, because on our servers we have a configuration about regex usernames and channel names, you need to get this setting from server and try to match.
To find this setting on your own server go to `Administration -> General -> UTF8`.
![Screen Shot 2020-02-08 at 14 48 30](https://user-images.githubusercontent.com/29778115/74089641-2d092780-4a82-11ea-8533-7a20bd55c6b3.png)

Reply:
Ok cool, thanks for letting me know about this. I will make the required changes.
:+1: